src/otx/core/data/entity/base.py

```diff
@@ -206,6 +207,143 @@ def __repr__(self) -> str:
         )
 
 
+class VideoInfo(tv_tensors.TVTensor):
+    """Meta info for image.
```

Comment:
```suggestion
    """Meta info for video.
```
It seems typo.

Reply:
fixed at b92e52520aed9a19150f81e80ea272b1e4135155